t/023-rewrite/duplicate.t

```diff
@@ -0,0 +1,350 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
I think the file name `dup-cmds.t` is clearer than `duplicate.t`. What's your opinion? ;)